lib/src/native/realm_bindings.dart

```diff
@@ -11119,12 +11244,13 @@ abstract class realm_sync_error_category {
   static const int RLM_SYNC_ERROR_CATEGORY_CLIENT = 0;
   static const int RLM_SYNC_ERROR_CATEGORY_CONNECTION = 1;
   static const int RLM_SYNC_ERROR_CATEGORY_SESSION = 2;
+  static const int RLM_SYNC_ERROR_CATEGORY_RESOLVE = 3;
```

Comment:
@nielsenko  I will create a new branch for adding this category in relation to #676